tests/integration/sync/test_sync_watch.py

```diff
@@ -91,6 +91,10 @@ def tearDown(self):
                 cfn_client.delete_stack(StackName=stack_name)
         super().tearDown()
 
+    def should_run_initial_infra_validation(self):
```

Comment:
Change this method to validate initial infra and override in other classes if needed to